content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,122 @@
+# plot templates
+
+Dump built-in plot templates to JSON files you can customize.
+
+
+## Synopsis
+
+```usage
+usage: dvc plots templates [-h] [-q | -v] [-o <path>]
+       [{simple,linear,confusion,confusion_normalized,scatter,smooth}]
+
+positional arguments:
+ TEMPLATE    Template to write. Writes all templates by default.
+```
+
+## Description
+
+Some times you may need to customize the way `dvc plots` are rendered beyond
+what the built-in [plot templates] allow.`dvc plots templates` can dump these templates
+to JSON files for you use as base for new templates. To modify them, use
+any valid elements of the [Vega-Lite specification].
+
+You can dump a specific built-in template by providing it's name as
+argument, for example `dvc plots templates confusion`. Template
+files are written to `.dvc/plots` by default, but any location can be
+set with the `--out` (`-o`) option.
+
+<admon type="note">
+
+Note that templates can only be used with
+[data-series plots](/doc/command-reference/plots#types-of-metrics).
+
+</admon>
+
+[plot templates]:
+  https://dvc.org/doc/command-reference/plots#plot-templates-data-series-only
+[Vega-Lite specification]: https://vega.github.io/vega-lite/
+  
+## Options
+
+- `-o <path>`, `--out <path>` - Directory to save templates to.
+
+## Example: Modifying simple template
+
+We will modify the `simple` template to show bars instead of line. Lets take
+sample `data.csv` file:
+
+```
+x,y
```

Comment:
```suggestion
## Example: Modifying the `simple` template

The built-in `simple` template can be an ideal base for custom templates
because it has a minimal structure you can make quick modifications to. For
example, let's show vertical bars instead of a connected line.

We'll work with the following `data.csv` file:

```csv
x,y
```